context.go

```diff
@@ -23,6 +44,16 @@ func newContext(c *Conode, o *Overlay, servID ServiceID, manager *serviceManager
 	}
 }
 
+// contextDataPath indicates where the service-data will be stored. If it is
+// empty, a memory-map is used.
+var contextDataPath = ""
+
+func init() {
+	if flag.Lookup("test.v") == nil {
```

Comment:
Mhhhhh... so what happens when a test is not ran with the verbose flag ?

I'm not a fan of this custom if here but yet again, I don't know of any other easy solutions.... :/